test/testutils.jl

```diff
@@ -9,11 +9,21 @@ using DataFlowTasks: R,W,RW
 # each block works for `s` seconds
 function fork_join(n,s)
     A = rand(2n)
-    @dspawn do_work(s) (A,) (RW,)
```

Comment:
Doing things like this is a bit more awkward with the new API

Reply:
This is a weird and somewhat artificial example since `A` never enters the actual computation, so I think it is fine that the API feels awkward here.